src/views/Home/components/Milestones/index.view.tsx

```diff
@@ -1,8 +1,39 @@
 import React from "react";
 import "./milestones.scss";
 
+const milestones = [
+  {
+    title: "Attendees",
+    text: "500+",
+  },
+  {
+    title: "First-time Hackers",
+    text: "47%",
+  },
+  {
+    title: "Projects",
+    text: "60+",
+  },
+  {
+    title: "In Prizes",
+    text: "$20,000+",
+  },
+];
+
 const MilestonesComponent: React.FC = () => {
-  return <div className="Milestones">Hello from MilestonesComponent</div>;
+  return (
+    <div className="Milestones">
+      <h1 className="Milestones__title">Milestones of 2020</h1>
+      <div className="Milestones__container">
+        {Object.entries(milestones).map(([_, milestone]) => (
```

Comment:
Since you created an array of objects, there's actually no need to do `map(([_, milestone])...`. You can simply just do 
```suggestion
        {Object.entries(milestones).map(({title, text}) => (
```
or
```suggestion
        {Object.entries(milestones).map((milestone) => (
```
I personally like de-structuring the objects since it creates slightly cleaner code when you go ahead and use the components/properties on lines 29-32